lesson09/index.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+function createListDom(itemList) {
+    const ulElement = document.getElementById("ul");
+    const frag = document.createDocumentFragment();
+
+    itemList.forEach(function(item) {
+        const liElement = document.createElement("li");
```

Comment:
Let's assume that we get the ul element globally.

1. A processing  in forEach to update as returning frag Fn.
2.   It use in ul appendChild args before `removeLoading()`.

By doing this,
created Fn by ① will be **used in anywhere** as long as  if it exists `itemList` JSON obj.
※ but, can set in `ul` element only

It's not the correct answer, so don't give too much weight

Please think this is an One of the ways. 
Sorry.

Reply:
Fixed the createListDom function by separating the roles

85a2015

https://codesandbox.io/s/modest-khorana-yicng?file=/index.js